builds/runtimes/pypy-5.3.1

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env bash
 # Build Path: /app/.heroku/python/
-# Build Deps: libraries/sqlite
 
 # NOTICE: This formula only works for the cedar-14 stack, not cedar.
 
 OUT_PREFIX=$1
 
+echo "Installing libsqlite3-dev"
+apt-get -qq update && apt-get install libsqlite3-dev
```

Comment:
don't you need super user permission here?

Reply:
It looks like the cedar-14 builder doesn't have super user since it's not using the container runtime.